controllers/topolvm_controller.go

```diff
@@ -216,7 +216,7 @@ func getControllerContainer() *corev1.Container {
 			},
 		},
 		LivenessProbe: &corev1.Probe{
-			Handler: corev1.Handler{
+			ProbeHandler: corev1.ProbeHandler{
```

Comment:
- are these due to the bump in k8sapi version number?

Reply:
Yes

Reply:
Please move any non-test file changes to a separate PR.

Reply:
done